### DISCUSSION TITLE
Disabling "XXX is disabled because of go1.18" warnings

### DISCUSSION THREAD

**Discussion by lzap** [golangci/golangci-lint#3218] [Q&A] [answered]
Guys,

is it possible to disable these warnings on the standard output (or stderr is it)? There are quite a bunch and it makes harder to scan with eyes to find actual errors. Thanks!

**Comment by bombsimon** [comment, marked as answer]
The easiest way would be to disable the linters in your config.

**Reply by lzap** [reply]
Ok works for me. I just upgraded to the latest release and most of them are gone! Great work.
